tests/php/bootstrap.php

```diff
@@ -118,6 +118,8 @@ function in_running_uninstall_group() {
 	return is_array( $argv ) && in_array( '--group=uninstall', $argv );
 }
 
+require dirname( __FILE__ ) . '/../../vendor/autoload.php';
```

Comment:
We should be able to use `__DIR__` here directly, since we no longer care about older PHP versions that didn't support it.

Reply:
Done in 34f7c850c5d03b48c82a4583f7e3daa6e837770c